upgrade_downgrade/test_dual_control_plane_upgrade_downgrade.sh

```diff
@@ -160,24 +160,24 @@ function verifyIstiod() {
 }
 
 kubectl rollout restart deployment echosrv-deployment-v1 -n "${TEST_NAMESPACE}"
-withRetries 30 10 checkDeploymentRolledOut "${TEST_NAMESPACE}" echosrv-deployment-v1
+withRetries 45 10 checkDeploymentRolledOut "${TEST_NAMESPACE}" echosrv-deployment-v1
 withRetries 5 20 verifyIstiod "${TEST_NAMESPACE}" "echosrv-deployment-v1" "v1" \
   "${TO_ISTIOCTL}" "istiod-${TO_REVISION}.istio-system.svc"
 withRetries 5 20 verifyIstiod "${TEST_NAMESPACE}" "echosrv-deployment-v2" "v2" \
   "${TO_ISTIOCTL}" "istiod.istio-system.svc"
 
 if [[ "${TEST_SCENARIO}" == "dual-control-plane-upgrade" ]]; then
   kubectl rollout restart deployment echosrv-deployment-v2 -n "${TEST_NAMESPACE}"
-  withRetries 30 10 checkDeploymentRolledOut "${TEST_NAMESPACE}" "echosrv-deployment-v2"
+  withRetries 45 10 checkDeploymentRolledOut "${TEST_NAMESPACE}" "echosrv-deployment-v2"
   withRetries 5 20 verifyIstiod "${TEST_NAMESPACE}" "echosrv-deployment-v2" "v2" \
     "${TO_ISTIOCTL}" "istiod-${TO_REVISION}.istio-system.svc"
   writeMsg "UPGRADE: Uninstall old version of control plane (${FROM_TAG})"
-  ${FROM_ISTIOCTL} experimental uninstall --filename -y "${FROM_PATH}/manifests/profiles/minimal.yaml"
+  ${TO_ISTIOCTL} experimental uninstall --filename "${FROM_PATH}/manifests/profiles/minimal.yaml" -y --force
```

Comment:
I'm confused on this part, could you write a README for dual control plane test? If this block of test is call "dual-control-plane-upgrade", why do you uninstall using the `TO_ISTIOCTL`?

Reply:
That was my attempt to fix uninstalling old control. It was failing because `istioctl experimental uninstall` is not supported in 1.6. You're right. I'll try to find another way - generating manifest and `kubectl delete` probably? 

Regarding how it was tested - I have already updated the README with my scenarios in the same directory. Let me know if it needs edits or something needs to be added.